jobs/scheduler/templates/application.properties.erb

```diff
@@ -38,6 +38,9 @@ client.ssl.trust-store=/var/vcap/data/certs/scalingengine/cacerts
 client.ssl.trust-store-password=123456
 client.ssl.protocol=TLSv1.2
 
+#Quartz
+org.quartz.scheduler.instanceName=app-autoscaler
```

Comment:
Ok, so the real problem we have is with instanceId. I would think that we want a stable instanceId across restarts. So the id should be based on the spec.id. You can see how diego does it for the rep's cell-id.